homeassistant/components/zwave/workaround.py

```diff
@@ -68,28 +68,19 @@
 
 def get_device_component_mapping(value):
     """Get mapping of value to another component."""
-    if (value.node.manufacturer_id.strip() and
-            value.node.product_type.strip()):
```

Comment:
Ah, I must have messed something up when I tested it in the interpreter. I'll restore it then and add a test.